src/main/java/com/volasoftware/tinder/controllers/AuthenticationController.java

```diff
@@ -49,4 +43,25 @@ public ResponseEntity<?> register(
                 HttpStatus.OK,
                 authenticationService.register(registerRequest));
     }
+
+    @ApiOperation(value = "Login Account")
+    @ApiResponses(
+            value = {
+                    @ApiResponse(responseCode = "200", description = "Successfully logged in!"),
+                    @ApiResponse(responseCode = "403", description = "Bad credentials!"),
+                    @ApiResponse(responseCode = "400", description = "Account not verified!"),
+                    @ApiResponse(responseCode = "404", description = "Account not found!")
+            }
+    )
+    @PostMapping(value = "/login",
+            consumes = {"application/xml", "application/json"})
+    public ResponseEntity<?> login(
+            @ApiParam(value = "Login details", required = true)
+            @RequestBody LoginRequest loginRequest) {
```

Comment:
Are you sure your validations work, because I don't see @Valid annotation for the LoginRequest parameter?